controllers/bsl.go

```diff
@@ -119,10 +119,16 @@ func (r *DPAReconciler) ReconcileBackupStorageLocations(log logr.Logger) (bool,
 		}
 		// Add the following labels to the bsl secret,
 		//	 1. oadpApi.OadpOperatorLabel: "True"
-		// 	 2. dataprotectionapplication.name: <name>
-		// which in turn will be used in the label handler to trigger the reconciliation loop
+		// 	 2. <namespace>.dataprotectionapplication: <name>
```

Comment:
this line change should be reverted, it was changed here https://github.com/openshift/oadp-operator/commit/71ea5b686962954b20874d0e03bc4387a537c09f

Reply:
done thanks!